cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -47,9 +50,15 @@
 PG_CONF_PATH = '/var/lib/pgsql/9.5/data/postgresql.conf'
 PGPASS_PATH = join(constants.CLOUDIFY_HOME_DIR, '.pgpass')
 
+CA_CERT_PATH = os.path.join(os.path.dirname(PG_CONF_PATH), 'root.crt')
+SERVER_CERT_PATH = os.path.join(os.path.dirname(PG_CONF_PATH), 'server.crt')
+SERVER_KEY_PATH = os.path.join(os.path.dirname(PG_CONF_PATH), 'server.key')
+
 PG_HBA_LISTEN_ALL_REGEX_PATTERN = r'host\s+all\s+all\s+0\.0\.0\.0\/0\s+trust'
+PG_HBA_HOSTSSL_REGEX_PATTERN = \
+    r'hostssl\s+all\s+all\s+0\.0\.0\.0\/0\s+md5\s+clientcert=1'
 
-PG_PORT = 5432
+PG_PORT = 15432 if config[POSTGRESQL_SERVER][SSL_ENABLED] else 5432
```

Comment:
Maybe put this in a func, so the port number will be set in one place and config will changed

Reply:
Fixed